libobs/obs-output.c

```diff
@@ -2129,6 +2129,14 @@ float obs_output_get_congestion(obs_output_t *output)
 	return 0;
 }
 
+int obs_output_get_bitrate_state(obs_output_t *output)
+{
+	if (!obs_output_valid(output, "obs_output_get_bitrate_state"))
+		return 0;
```

Comment:
Style: Could use a blank newline between this line and the next line.

Reply:
line added, dunno why github keeps this open